src/renderer/components/swap/Swap.tsx

```diff
@@ -895,16 +896,21 @@ export const Swap = ({
     [oQuote]
   )
   // Swap streaming result from thornode
-  const totalSwapSeconds: number = useMemo(
+  const transactionTime: SwapTime = useMemo(
     () =>
       FP.pipe(
         sequenceTOption(oQuote),
         O.fold(
-          () => 0,
-          ([txDetails]) => txDetails.txEstimate.totalSwapSeconds
+          () => ({ inbound: 0, outbound: 0, totalSwap: 0, streaming: 0 }),
+          ([txDetails]) => ({
+            inbound: isRuneAsset(sourceAsset, network) ? 6 : txDetails.txEstimate.inboundConfirmationSeconds, // Replace with actual value from txDetails
```

Comment:
Can 6 be a constant please. 
Blocktime can change. E.g. it use to be 5.

Reply:
waiting on the latest thorchain-query to support the const here